spec/features/merchants/dashboard/index_spec.rb

```diff
@@ -12,4 +12,19 @@
       expect(page).to have_content("Barry's Dashboard")
     end
   end
-end
\ No newline at end of file
+
+  describe "User Story 2 - Merchant Dashboard Links" do
+    it "has links to Merchant's items index and invoices index" do
+      expect(page).to have_link("Barry's Items")
+
+      click_link("Barry's Items")
+      expect(page.current_path).to eq("/merchants/#{@merchant_1.id}/items")
```

Comment:
You can put `current_path` instead of `page.current_path` if you want. It's just whichever you prefer.